pkg/transformers/transformers.go

```diff
@@ -0,0 +1,79 @@
+package transformers
```

Comment:
All transforms are done clientside as API requests with a `name_transformer` parameter aren't able to be cached.